v8js_class.cc

```diff
@@ -270,6 +270,8 @@ static void v8js_jsext_free_storage(v8js_jsext *jsext) /* {{{ */
 		v8js_free_ext_strarr(jsext->deps, jsext->deps_count);
 	}
 	delete jsext->extension;
+	// jsext->name and jsext->source are interned, this would do nothing. 
+	// Interned strings will be freed after module shutdown (On ZTS - But on NTS, they're actually interned)
```

Comment:
I think this comment is (now) misleading/wrong, since the strings are guaranteed *not* to be interned